javalin/src/test/java/io/javalin/TestResult.kt

```diff
@@ -40,26 +40,6 @@ internal class TestResult {
         assertThat(http.getBody("/errored")).isEqualTo(INTERNAL_SERVER_ERROR.message)
     }
 
-    @Test
-    fun `result function can be called once per handler`() = TestUtil.test { app, http ->
```

Comment:
This test is needed to verify if result method can be called only once per scope in general, callback is just one of them here

Reply:
I don't understand how this test proves that 🤔